07_hello_friend/friend.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Friend
+  def greeting(who = nil)
+    if who.nil?
```

Comment:
try to avoid cascade `if` statements (that's not critical at all, but ..
```
return 'Hello!' if who.blank?

"Hello, #{who}!"
```
looks much prettier).